### ISSUE TITLE
I can't install 18.03.1-ee-2

### ISSUE THREAD

**Issue by sixcorners** [OneGet/MicrosoftDockerProvider#44]
```powershell
PS C:\Users\Administrator> install-package docker -update -provider dockermsftprovider -requiredversion 18.03.1-ee-2 -ma
ximumversion 18.03.1-ee-2 -allversions

The package(s) come(s) from a package source that is not marked as trusted.
Are you sure you want to install software from 'DockerDefault'?
[Y] Yes  [A] Yes to All  [N] No  [L] No to All  [S] Suspend  [?] Help (default is "N"): y
WARNING: A restart is required to enable the containers feature. Please restart your machine.

Name                           Version          Source           Summary
----                           -------          ------           -------
Docker                         18.03.1-ee-2     DockerDefault    Contains Docker EE for use with Windows Server.


PS C:\Users\Administrator> get-package docker

Name                           Version          Source                           ProviderName
----                           -------          ------                           ------------
docker                         17.06.2-ee-15    DockerDefault                    DockerMsftProvider


PS C:\Users\Administrator>
```
Am I doing something wrong?

**Comment by carlfischer1** [comment]
@sixcorners no. there is a final update to the package list required now that 18.03.1-ee-2 is available

ping @cwilhit

**Comment by cwilhit** [comment]
Thanks, it was a problem on our end. We've resolved it and I've verified on my machine.

**Comment by sixcorners** [comment]
So I kind of forget where I got the 18.03.1-ee-2 version identifier from (I think it was from an error message) but the latest version [here](https://docs.docker.com/ee/engine/release-notes/) is "18.03.1-ee-1 (2018-06-27)". Also the summary on the 18.03.1-ee-2 that I just installed is "This is a test". Is 18.03.1-ee-2 supposed to be available? I think maybe I should have been trying to install 18.03.1-ee-1.
